src/device-registry/models/Site.js

```diff
@@ -414,7 +414,6 @@ siteSchema.statics = {
           "airqlouds.location": 0,
           "airqlouds.airqloud_tags": 0,
           "airqlouds.long_name": 0,
-          "airqlouds.createdAt": 0,
```

Comment:
enabled the createdAt field when getting Airqloud details with the get Sites endpoint. cc: @ja-odur